pom.xml

```diff
@@ -458,6 +458,45 @@
                 <module>delta-lake/delta-22x</module>
             </modules>
         </profile>
+        <profile>
+            <id>release332cdh</id>
+            <activation>
+                <property>
+                    <name>buildver</name>
+                    <value>332cdh</value>
+                </property>
+            </activation>
+            <properties>
+                <buildver>332cdh</buildver>
+                <spark.version>${spark332cdh.version}</spark.version>
+                <spark.test.version>${spark332cdh.version}</spark.test.version>
+                <parquet.hadoop.version>1.10.99.7.1.9.0-387</parquet.hadoop.version>
+                <iceberg.version>${spark330.iceberg.version}</iceberg.version>
+            </properties>
+            <repositories>
+                <repository>
+                    <id>cloudera-repo</id>
+                    <url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>
+                    <releases>
+                        <enabled>true</enabled>
+                    </releases>
+                    <snapshots>
+                        <enabled>true</enabled>
+                    </snapshots>
+                </repository>
+            </repositories>
+            <pluginRepositories>
+                <pluginRepository>
+                    <id>cloudera-repo</id>
+                    <url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>
+                </pluginRepository>
+            </pluginRepositories>
```

Comment:
We get more and more of these. Let us move repeated repo definitions out of all CDH profiles and have them defined in the global lists at the bottom of the pom like this:

```xml
  <repositories>
      ...
      <repository>
            <id>cloudera-repo</id>
            <url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>
            <releases>
                <enabled>${cloudera.repo.enabled}</enabled>
            </releases>
            <snapshots>
                <enabled>${cloudera.repo.enabled}</enabled>
            </snapshots>
        </repository>
  </repositories>
  <pluginRepositories>
...
        <pluginRepository>
            <id>cloudera-repo</id>
            <url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>
            <releases>
                <enabled>${cloudera.repo.enabled}</enabled>
            </releases>
            <snapshots>
                <enabled>${cloudera.repo.enabled}</enabled>
            </snapshots>
        </pluginRepository>
   </pluginRepositories>
```

the we can add just a single line to each CDH profile properties:
```xml
<cloudera.repo.enabled>true</cloudera.repo.enabled>
```